### ISSUE TITLE
Suggestion to clarify the manual

### ISSUE THREAD

**Issue by bertulli** [moderncv/moderncv#132]
Hi all!
I was using this package and I noticed the manual is quite unclear, almost misleading about the usage of `\cventry`. It is said (section 3.2): "in the sections describing your education or your job experiences, you can use:
```latex
\cventry{years}{degree/job title}{institution/employer}
{localization}{optional: grade/...}
{optional: comment/job description}
```
where the last three arguments are optional"

The point is that the arguments are **not** actually optional, they are mandatory, but they can be left empty. If I try to use them as optional (not providing them), I get errors, as you can see in [this TeX SE question](https://tex.stackexchange.com/questions/656044/moderncv-package-provides-error-for-optional-arguments-why).

Maybe the documentation can be reworded? I can do a PR if you like it.

**Comment by stephanlachnit** [comment]
> Maybe the documentation can be reworded? I can do a PR if you like it.

Yes, the documentation needs to be reworked. There was some previous work at https://github.com/moderncv/moderncv/pull/58. Feel free to give it a try!

**Comment by bertulli** [comment]
I'd love so, but I think the repo misses some files. For instance, when
I try to compose the file `moderncv_userguide.tex`, I get errors about a
missing file. And actually, on line 208 I can see
```latex
 \fbox{\includegraphics[angle=90,width=\textwidth]{cv_casual.pdf}}
```
What am I doing wrong?

However, I made the "basic" PR, meaning that I only changed the unclear
part of the manual (line 286). Is it ok?

**Comment by ShadowMitia** [comment]
I haven't built the userguide yet, but I think you are missing some examples that are compiled?
You probably need this PR https://github.com/moderncv/moderncv/pull/54 to get the Makefile which should help you generate everything (hopefully).

**Comment by stephanlachnit** [comment]
> However, I made the "basic" PR, meaning that I only changed the unclear
part of the manual (line 286). Is it ok?

Yes, thanks! Closed via #133
